github/actions/client.go

```diff
@@ -892,7 +917,7 @@ func (c *Client) getActionsServiceAdminConnection(ctx context.Context, rt *regis
 
 	c.logger.Info("getting Actions tenant URL and JWT", "registrationURL", registrationURL.String())
 
-	resp, err := http.DefaultClient.Do(req)
+	resp, err := c.Do(req)
```

Comment:
there is another `http.DefaultClient` in the `client.go` you might want to replace with `c.Do`

Reply:
Thank you!